dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -830,6 +830,22 @@ public void rename(String src, String dst, RenamePOptions options)
     }
 
     try {
+      boolean overWrite = options.hasS3SyntaxOptions()
+          && options.getS3SyntaxOptions().hasOverwrite()
+          && options.getS3SyntaxOptions().getOverwrite();
+      if (dstUfs.exists(dst)) {
+        if (!overWrite) {
+          throw new RuntimeException(
```

Comment:
why it should be a RuntimeException, just the FileAlreadyExitsException is enough.